source/docs/software/wpilib-tools/robot-simulation/index.rst

```diff
@@ -2,9 +2,13 @@ Robot Simulation
 ================
 
 .. toctree::
-   :maxdepth: 1
```

Comment:
I don't agree with this maxdepth change. Maxdepth2 really clutters content.

Reply:
Why is Glass maxdepth 2?  I was just trying to standardize.  I don't care what we standardize on but don't do 2 extremely similar things differently for no reason

Reply:
It's a sort of mixture of feelings of mine. Generally, I only use maxdepth for showing nested articles. Maxdepth of 2 when there is no nested articles shows sections inside those articles, which I don't necessarily agree on, as it can quickly clutter the TOC. I'm going to say both should be 1.